rpc-server/src/dispatchers/mempool.rs

```diff
@@ -35,9 +38,9 @@ impl MempoolInterface for MempoolDispatcher {
         let tx = Transaction::deserialize_from_vec(&hex::decode(&raw_tx)?)?;
         let txid = tx.hash::<Blake2bHash>();
 
-        match self.mempool.add_transaction(tx, None) {
+        match self.consensus.send_transaction(tx).await {
```

Comment:
Shouldn't it be both, add to own mempool and send the transaction over the network?